### ISSUE TITLE
PlugInstall fails

### ISSUE THREAD

**Issue by EricEWeir** [junegunn/vim-plug#1179]
PlugInstall fails. I get the following error message:

    xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

------------------------------

```
I do not know how to get output of ":version" into the clipboard.
```

- Type:
    - [ ] Bug
    - [ ] Enhancement
    - [ ] Feature Request
    - [x] Question
- OS:
    - [ ] All/Other
    - [ ] Linux
    - [x] OS X
    - [ ] Windows
- Vim:
    - [ ] Terminal Vim
    - [x] GVim  (MacVim?)
    - [ ] Neovim

**Comment by EricEWeir** [comment]
I am not a coder. Just a writer who’s a minimally competent vim user, thanks to people on the vim-use and mac-vim listservs. I haven’t the faintest idea how to go about analyzing/fixing this problem. I would appreciate any help anyone might be able to give.

**Comment by junegunn** [comment]
You should investigate why you're getting the error message. (i.e. Google "xcrun: error: invalid active developer path")
It's not from vim-plug itself.

**Comment by EricEWeir** [comment]
> On May 12, 2022, at 10:17 PM, Junegunn Choi ***@***.***> wrote:
> 
> You should investigate why you're getting the error message. (i.e. Google "xcrun: error: invalid active developer path")
> It's not from vim-plug itself.
> 
Thanks greatly. No way I could known this or figured it out. I’ve checked it out. Solution looks straightforward, though I gather it is not always. Keeping my fingers crossed.

Sincerely,
———————————
Eric Weir 
Atlanta, GA  USA

“It has all been combustion.”

- W.G. Sebald

**Comment by EricEWeir** [comment]
I got CommandLineTools installed, but PlugInstall is still not working. It shows "vim-plug: Already installed" but doesn't install the plugins in my plugin list.

I note the ruby and python requirements. Doing "version" shows +python/dyn, python3/dyn, and +ruby/dyn.

Searching the web for how to tell if vim has python support I found this: "python import sys; print(sys.version)" Using it I got "2.7.16 (default, Sep  6 2021, 07:39:44)"

Doing likewise for ruby I found this: "echo has('ruby')" Using I got "0".

So, is the situation regarding ruby possibly why PlugInstall is not working for me? If so, what can I do about it?
